.landscaper/gardenlet/component_descriptor_list.yaml

```diff
@@ -0,0 +1,28 @@
+# Component descriptor list containing the image reference for the Gardenlet deployment.
```

Comment:
@schrodit and colleagues, I guess this is only an example file of how a component descriptor looks like and has no  value in the actual execution of this landscaper component, correct? If not, how will it be maintained?

Reply:
I moved it to the example folder. 
This way a locally executed Landscaper component can use it.